### ISSUE TITLE
use pre-trained BERT models for downstream tasks

### ISSUE THREAD

**Issue by stellaywu** [NVIDIA/NeMo#341]
Wondering if it's possible to use the pre-trained BERT model/weights from the checkpoints with huggingface library for downstream tasks?

**Comment by ekmb** [comment]
yes, you can find such examples here https://github.com/NVIDIA/NeMo/tree/master/examples/nlp

**Comment by stellaywu** [comment]
thanks for the reply! A follow up question, I am trying to do multi-label classification with self-pretrained BERT checkpoints. How to pass multi label data to data layer? Suppose my data_file is a csv file with a data column and N label column. 
`
data_layer = nemo.collections.nlp.nm.data_layers.text_classification_datalayer.BertSentenceClassificationDataLayer(
        input_file=data_file,
        tokenizer=tokenizer,
        max_seq_length=args.max_seq_length,
        num_samples=num_samples,
        shuffle=shuffle,
        batch_size=batch_size,
    )
`
Thanks!

**Comment by yzhang123** [comment]
check out https://github.com/NVIDIA/NeMo/blob/master/nemo/collections/nlp/data/datasets/text_classification_dataset.py#L50-L52 for the input file format. Basically, you have 2 columns: the first is the sentence, the second is the class label as an integer.

**Comment by yzhang123** [comment]
@stellaywu  can this issue be closed?

**Comment by stellaywu** [comment]
@yzhang123  thanks for the response. Looks like text_classification_dataset.py doesn't take multilable int he second column?

**Comment by ekmb** [comment]
@stellaywu the example can handle multilable tasks. Please see this example of the datapreprocessing: https://github.com/NVIDIA/NeMo/blob/master/nemo/collections/nlp/data/datasets/text_classification_dataset.py#L253

**Comment by stellaywu** [comment]
Thanks!
